waiter/resources/log4j.properties

```diff
@@ -18,3 +18,13 @@ log4j.appender.ErrorAppender.DatePattern='.'yyyy-MM-dd
 log4j.appender.ErrorAppender.layout=org.apache.log4j.PatternLayout
 # CID will be replaced by the custom pattern layout configured in waiter.correlation-id/replace-pattern-layout-in-log4j-appenders
 log4j.appender.ErrorAppender.layout.ConversionPattern=%d{ISO8601} %-5p %c [%t] - [CID] %m%n
+
+log4j.category.waiter.request-log=INFO, requestlog
```

Comment:
we should have a comment in here about why this is configured to be separate.
It also means we cannot have any other log statements in `request-log.clj`